src/core/server/app/handlers/api/auth/local/signup.ts

```diff
@@ -0,0 +1,113 @@
+import Joi from "joi";
+
+import { AppOptions } from "talk-server/app";
+import { handleSuccessfulLogin } from "talk-server/app/middleware/passport";
+import { validate } from "talk-server/app/request/body";
+import { RequestLimiter } from "talk-server/app/request/limiter";
+import { IntegrationDisabled } from "talk-server/errors";
+import { GQLUSER_ROLE } from "talk-server/graph/tenant/schema/__generated__/types";
+import { LocalProfile, User } from "talk-server/models/user";
+import { insert } from "talk-server/services/users";
+import { sendConfirmationEmail } from "talk-server/services/users/auth";
+import { RequestHandler } from "talk-server/types/express";
+
+export interface SignupBody {
```

Comment:
Only issue I see with this is the ramifications of unifying that naming.. Camelcase version makes sense, but for routes, it would have to become `/api/auth/local/sign-up`. Not 100% sold.

Reply:
Just the `interface` then maybe? I mean we also have `GraphQL` interfaces but still call the endpoint in the route `graphql` (maybe not the best example but you get the point)